### DISCUSSION TITLE
How do I get node pool information for my cluster?

### DISCUSSION THREAD

**Discussion by Vikas4790** [kubernetes-client/csharp#1271] [Q&A] [answered]
I could not find the client library having a method to report the node pool or list of node pools. 
I was trying to get information regarding node pools like the native aks cli command provides - 

`az aks nodepool show --cluster-name myCluster -g myClusterResourceGroup --name myNodePool`

**Comment by tg123** [comment, marked as answer]
this is not something the sdk covers
howerver, you can query the node label to get which nodepool the node is

**Reply by Vikas4790** [reply]
Yaa, that's what I am doing right now. I can extract the node pool name by looking at the node labels. However, I wanted to get further information about the node pool like the auto-scaling limits etc, but seems like I can't do it using the sdk as of now. Thanks for the answer :)

**Reply by koureasstavros** [reply]
I confirm that using this SDK you cannot pull information about the node pool like "Scaling Method" or "Minimum node count" and "Maximum node count". This would be very benefical to the sdk, as more automation tasks can be archieved.

**Reply by tg123** [reply]
you need aks sdk not k8s sdk in this case
